specs/deneb/p2p-interface.md

```diff
@@ -187,6 +187,16 @@ The following validations MUST pass before forwarding the `blob_sidecar` on the
 - _[REJECT]_ The sidecar is proposed by the expected `proposer_index` for the block's slot in the context of the current shuffling (defined by `block_header.parent_root`/`block_header.slot`).
   If the `proposer_index` cannot immediately be verified against the expected shuffling, the sidecar MAY be queued for later processing while proposers for the block's branch are calculated -- in such a case _do not_ `REJECT`, instead `IGNORE` this message.
 
+The gossip `ForkDigest`-context is determined based on `compute_fork_version(compute_epoch_at_slot(blob_sidecar.signed_block_header.message.slot))`.
+
+Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
+
+[0]: # (eth2spec: skip)
+
+| `fork_version`                 | Chunk SSZ type                |
+|--------------------------------|-------------------------------|
+| `DENEB_FORK_VERSION` and later | `deneb.BlobSidecar`           |
+
```

Comment:
The `ForkDigest`-context thing is only for Req/Resp, not GossipSub topics. GossipSub has nothing to do with it.

Reply:
`ForkDigest` is also part of Gossipsub (as part of the topic name): https://github.com/ethereum/consensus-specs/blob/a42d6706d89c414764eda7e2d0103e19f1e23761/specs/phase0/p2p-interface.md#topics-and-messages

Reply:
Let's call it `ForkDigestValue` then.